frontend/src/scenes/events/eventsTableLogic.js

```diff
@@ -182,6 +199,18 @@ export const eventsTableLogic = kea({
                 { replace: true },
             ]
         },
+        toggleAutomaticLoad: () => {
+            return [
+                router.values.location.pathname,
+                {
+                    ...router.values.searchParams,
+                    properties: values.properties,
+                    autoload: values.automaticLoadEnabled,
+                },
+                window.location.hash,
```

Comment:
This will work, but sometimes there's no `window` and then this could be replaced with `router.values.hashParams`.